studio/stores/pgmeta/MetaStore.ts

```diff
@@ -265,6 +265,46 @@ export default class MetaStore implements IMetaStore {
     return await this.query(query)
   }
 
+  async updateTableRealtime(table: PostgresTable, enable: boolean) {
+    const publications = this.publications.list()
+    const publicTables = this.tables.list((table: PostgresTable) => table.schema === 'public')
+
+    const realtimePublication = publications.find((pub) => pub.name === 'supabase_realtime')
+    const { id, tables: publicationTables } = realtimePublication
```

Comment:
Is this destructuring safe to do here? What if it doesn't find the `supabase_realtime` publication?

Reply:
oh yeah you're right actually - lemme push a small update to make it a bit more fault resistant